setup.py

```diff
@@ -1,8 +1,11 @@
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
 setup(
-    name='scrapy-dropbox',
-    packages=['scrapy_dropbox'],
-    install_requires=['scrapy', 'dropbox'],
-    requires=['scrapy', 'dropbox']
+    name="scrapy-dropbox",
```

Comment:
I wonder if we should rename this in line with the repository name. No strong opinion, though.